src/MessagingThreadHistory/MessagingThreadHistory.tsx

```diff
@@ -20,12 +20,14 @@ export interface MessageData {
   timestamp?: Date;
   content: React.ReactNode;
   index: number;
+  isRead?: boolean;
 }
 
 interface Props {
   className?: string;
   threadID: string;
   messages: MessageData[];
+  showReadStatus?: boolean;
```

Comment:
Just curious how necessary this prop is. If the consumer doesn't want read status to be displayed, they could just omit the `isRead` prop on the `messages`.

Reply:
Ah, not super necessary. I think I initially wanted to make it difficult to have only a portion of the messages to show the read status but the consumer should probably make sure of that. Removing the field!